CONTRIBUTING.rst

```diff
@@ -47,13 +46,13 @@ For other questions, discussions, and chats, we have:
   for scancode and other related tools. You can use your
   favorite IRC client or use the web chat at https://webchat.freenode.net/
 
-- a mailing list at https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss
+- a mailing list at `sourceforge <https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss>`_
 
 
 Bug reports
 ===========
 
-When reporting a bug at https://github.com/nexb/scancode-toolkit/issues please
+When reporting a bug at `issues <https://github.com/nexb/scancode-toolkit/issues>`_ please
```

Comment:
Same as above: "reporting a bug" could be the link text.

Reply:
sure